modules/geoip2/tests/test_geoip_parser.c

```diff
@@ -89,6 +89,15 @@ parse_geoip_into_log_message(const gchar *template_format)
   return msg;
 }
 
+Test(geoip2, template_is_mandatory)
+{
+  LogParser *geoip2_parser = maxminddb_parser_new(configuration);
+
+  cr_assert_not(log_pipe_init(&geoip2_parser->super));
```

Comment:
If you want to be sure, you could use message grabbing.

Reply:
With this test I want to protect against ensuring the `geoip2` only can start if there is a template, not that there is an exact error message about it. As this test could do just that, I tend to go with kiss here.